documentation/src/main/asciidoc/topics/productized/spring_boot_starter.adoc

```diff
@@ -0,0 +1,272 @@
+=== Using the Spring Boot Starter
+Quickly get your Spring Boot project up and running with {brandname}.
+
+The {brandname} starter provides a set of managed transitive dependencies that include everything your Spring Boot project needs to seamlessly interact with {brandname} in one of two modes:
+
+* Embedded Mode runs {brandname} inside your application.
+* Remote Client/Server Mode connects your application to an {brandname} cluster.
+
+[TIP]
+====
+The {brandname} Spring Boot starter gives you a convenient way to get started with Spring Boot but is optional. To use {brandname} with Spring Boot you can simply add the dependencies you want.
+====
+
+==== Setting Up Your Project
+
+[NOTE]
+====
+{brandname} supports Spring Boot 1.5.x and 2.0. For Spring Boot 1.5.x, use the Spring 4 dependencies. For Spring Boot 2.0, use the Spring 5 dependencies.
```

Comment:
Here is a separate PR for 9.4.x: https://github.com/infinispan/infinispan/pull/6568 I've reworked this one for 10 because there are still some changes needed to those docs.